src/forms/userForm.tsx

```diff
@@ -0,0 +1,372 @@
+                    let updatedPermissions: UserPermissionsObject;
+                    updatedPermissions = {
+                      model_management: this.state.permissions.model_management,
+                      asset_management: this.state.permissions.asset_management,
+                      power_control: this.state.permissions.power_control,
+                      audit_read: this.state.permissions.audit_read,
+                      admin: !this.state.permissions.admin,
+                      datacenter_permissions: this.state.permissions
+                        .datacenter_permissions
+                    };
+                    this.setState({
+                      permissions: updatedPermissions
+                    });
+                  }}
+                ></Checkbox>
+              </FormGroup>
+              <FormGroup inline={true}>
+                <Checkbox
+                  label="Model Management"
+                  alignIndicator={Alignment.LEFT}
+                  checked={
+                    this.state.permissions.admin
+                      ? true
+                      : this.state.permissions.model_management
+                  }
+                  disabled={this.state.permissions.admin}
+                  onChange={() => {
+                    this.setState({
+                      permissions: this.updateBooleanPermissions(
+                        "model_management"
+                      )
+                    });
+                  }}
+                ></Checkbox>
+              </FormGroup>
+              <FormGroup inline={true}>
+                <Checkbox
+                  label="Power Control"
+                  alignIndicator={Alignment.LEFT}
+                  checked={
+                    this.state.permissions.admin
+                      ? true
+                      : this.state.permissions.power_control
+                  }
+                  disabled={this.state.permissions.admin}
+                  onChange={() => {
+                    this.setState({
+                      permissions: this.updateBooleanPermissions(
+                        "power_control"
+                      )
+                    });
+                  }}
+                ></Checkbox>
+              </FormGroup>
+              <FormGroup inline={true}>
+                <Checkbox
+                  label="Audit Log"
+                  checked={
+                    this.state.permissions.admin
+                      ? true
+                      : this.state.permissions.audit_read
+                  }
+                  alignIndicator={Alignment.LEFT}
+                  disabled={this.state.permissions.admin}
+                  onChange={() => {
+                    this.setState({
+                      permissions: this.updateBooleanPermissions("audit_read")
+                    });
+                  }}
+                ></Checkbox>
+              </FormGroup>
+              <RadioGroup
```

Comment:
ideally if administrator permission is checked, all Asset Management editing should be disabled too

is it possible to add another Check box next to Asset Management (currently "Datacenter permissions")? if I wanted to remove asset permission from a user, I don't think it's clear enough that you have to toggle to Per Datacenter and not select anything

Reply:
another bug - selecting the Global radio button & submitting does not set asset_management correctly, it sends:
```
id: 8
model_management: false
asset_management: false
power_control: false
audit_read: true
admin: false
datacenter_permissions: []
```
(I had audit log checked so that one's right)